benchmarkapps/BasicApi/BasicApi.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
What fails if we use `$(StandardTestWebsiteTfms)` in these projects with hard-coded target frameworks? If we can, would be great to get rid of the special cases.

Reply:
The variable isn't defined in the root Directory.Build.props (it's in the test Directory.Build.props). So wouldn't be available here. In addition, I think these projects are meant to be buildable standalone on the benchmark server, so I'm going to leave it be for now

Reply:
Ah, now I remember doing that&hellip;